admin/system/insert-produk.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    //get data
+    $nama_produk = $_POST["nama_produk"];
+    $jenis_produk = $_POST["jenis_produk"];
+    $id = uniqid();
+    mkdir("../upload/".$id);
```

Comment:
Codacy found an issue: [The use of function mkdir() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)